frontend/app/.server/domain/services/applicant.service.ts

```diff
@@ -58,8 +58,8 @@ export class DefaultApplicantService implements ApplicantService {
     }
 
     const clientNumber = this.applicantDtoMapper.mapApplicantResponseEntityToClientNumber(applicantResponseEntity.unwrap());
-    this.log.trace('Returning client number [%s] for sin [%s]', clientNumber, sin);
-
-    return clientNumber ? Some(clientNumber) : None;
+    if (clientNumber.isNone()) this.log.trace('Client number for sin [%s] not found; Returning null', sin);
+    else this.log.trace('Returning client number [%s] for sin [%s]', clientNumber.unwrap(), sin);
```

Comment:
Use `unwrapUnchecked` to returns the contained `Some` value or undefined if `None`

```suggestion
    this.log.trace('Returning client number [%s] for sin [%s]', clientNumber.unwrapUnchecked(), sin);
```